web/src/components/tasks/anytime-action/tax-clearance-certificate/steps/CheckEligibility.tsx

```diff
@@ -21,7 +20,7 @@ interface Props {
   saveTaxClearanceCertificateData: () => void;
 }
 
-export const TaxClearanceStepTwo = (props: Props): ReactElement => {
+export const CheckEligibility = (props: Props): ReactElement => {
```

Comment:
These step components are already in a location that makes it obvious they are for Tax Clearance. Dropped the lengthier name, as this isn't really giving us any value, in favor of a shorter, more semantically accurate name.